src/middlewared/middlewared/plugins/crypto.py

```diff
@@ -2018,10 +2529,11 @@ def set_enum(attr):
         Int('id', required=True),
         Dict(
             'ca_update',
+            Bool('revoked'),
             Int('ca_id'),
             Int('csr_cert_id'),
             Str('create_type', enum=['CA_SIGN_CSR']),
-            Str('name'),
+            Str('name')
```

Comment:
Nitpick, in our codebase we prefer to end things with a comma, that way adding a new entry does not require 2 line changes.